### ISSUE TITLE
Issue with showing up custom moons.

### ISSUE THREAD

**Issue by Uniteria** [HolographicWings/LethalExpansion#109]
Hi, when changing setting from Lethalexpansion.core to normal Lethalexpansion it doesn't show several custom moons on terminal.

**Comment by HolographicWings** [comment]
LECore is not my mod
I dunno how to help you I'm sorry

**Comment by HolographicWings** [comment]
LECore is not my mod
What setting are you talking about ?

**Comment by JerryLzysss** [comment]
maybe you should delete the original LethalexpansionCore, or if you are using the LethalExpansionCore written by jockie, here are the setting. 
> UseOriginalLethalExpansion: Whether or not to use the original LethalExpansion instead of LethalExpansionCore when they are both loaded, default false
I used to be frustrated about the problem until i find the solution.

**Comment by JerryLzysss** [comment]
maybe you should delete the original LethalexpansionCore, or if you are using the LethalExpansionCore written by jockie, here are the setting.
I used to be frustrated about the problem until i find the solution.
> 
UseOriginalLethalExpansion: Whether or not to use the original LethalExpansion instead of LethalExpansionCore when they are both loaded, default false

**Comment by JerryLzysss** [comment]
maybe you should delete the original LethalexpansionCore, or if you are using the LethalExpansionCore written by jockie, here are the setting.
I used to be frustrated about the problem until i find the solution.

> UseOriginalLethalExpansion: Whether or not to use the original LethalExpansion instead of LethalExpansionCore when they are both loaded, default false

**Comment by a1russell** [comment]
> LECore is not my mod

I am guessing the OP wasn't being clear. Some mods depend on LethalExpansionCore instead of LethalExpansion. LethalExpansionCore by default suppresses use of your mod, LethalExpansion. There is a config to enable use of LethalExpansion, which is your mod, instead of LethalExpansionCore.

It may or may not be relevant to the issue, but I believe OP is saying that they are facing an issue with LethalExpansion itself after toggling that config so that LethalExpansionCore is no longer used.

The issue is that most custom moons no longer show on the terminal. Personally, I am experiencing this myself, as well as several other problems (terminal not clearing, unable to route to moons, items not appearing on ship). I'm not sure which mod is at fault for all of these though. Seems to be either LethalExpansion or LethalLevelLoader.

-------

> if you are using the LethalExpansionCore written by jockie, here are the setting

I am pretty sure the OP already toggled that setting to true.

**Comment by Uniteria** [comment]
Right, the issue was LethalLevelLoader which seems to be fixed, but prices now on custom modded moons are 0 :(
